app/components/Form/AmountInput.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import CurrencyIcon from 'components/CurrencyIcon'
+import styles from './RequestForm.scss'
+
+class AmountInput extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.handleChange = this.handleChange.bind(this)
+    this.handleKeyDown = this.handleKeyDown.bind(this)
+  }
+
+  handleChange(e) {
+    let value = e.target.value;
+    let integer = null;
+    let fractional = null;
+
+    if (value.match(/[0-9]+/)) {
+      integer = e.target.value * 1;
+    }
+    if (value.match(/[0-9]*[,\.][0-9]+/)) {
+      [integer, fractional] = value.toString().split('.');
```

Comment:
The split could be a `.` or ` `,` as per the regex.

Reply:
fixed